### ISSUE TITLE
newImageFont assigns unchangeable 1 pixel of extra spacing

### ISSUE THREAD

**Issue by ericabxy** [libretro/libretro-lutro#234]
Fonts created with love.graphics.newImageFont automatically have 1 pixel of extra spacing. This is different from the default behavior of LÖVE which is 0 extra spacing. Furthermore, Lutro does not recognize the extraspacing argument, so there is no way for content to correct this behavior.

**Comment by RobLoach** [comment]
Could you provide a test for this?

**Comment by EricxDu** [comment]
Yes. I wrote a test using a font meant for uniform -1 kerning, demonstrating the results of constructing the font with several different "extraspacing" arguments.

[extraspacing.zip](https://github.com/libretro/libretro-lutro/files/8802167/extraspacing.zip)

**Comment by kivutar** [comment]
You're right, I need to fix this.

**Comment by EricxDu** [comment]
This issue appears to have been fixed by [https://github.com/libretro/libretro-lutro/commit/e9395ab4549b74670598d811ca8094be513c4c5c](url)
Running the test file with the latest code using Lutro and LÖVE results in the same behavior in all four test cases.
I'm trying to upload a slightly updated test file in case anyone else needs to test.

**Comment by EricxDu** [comment]
[extraspacing.zip](https://github.com/libretro/libretro-lutro/files/9823319/extraspacing.zip)
Here is the new test file. It has no subfolder so be careful when extracting, or simply rename it to extraspacing.love or extraspacing.lutro to use it.
